src/Controls/StreetControls.js

```diff
@@ -310,6 +310,8 @@ class StreetControls extends FirstPersonControls {
             })
             .start();
 
+        this.tweenGroup.add(this.tween);
+
```

Comment:
From `tween.js` [changelog](https://github.com/tweenjs/tween.js/releases/tag/v24.0.0):

> Tweens are no longer automatically added or removed from groups by default when you call any Tween methods such as start(), stop(), or pause(), and the preserve parameter to Group.update() now defaults to true and is deprecated to be removed in a future major version.

As such, tweens are not removed in the group and updated continuously (they were automatically removed before the update).

Reply:
Bien vu; done :)